espanol/dragon/dragon.md

```diff
@@ -1,3 +1,21 @@
 Para iniciar la batalla, tenía que conseguir un medio de transporte.
-Y este medio de transporte sería el Dragón Earhart, que compró al más famoso 
```

Comment:
Unfortunately, original file do not end with an EOL character, so it's impossible to add anything without modificate the last line.